src/components/editor-page/document-bar/share/share-modal.tsx

```diff
@@ -24,18 +24,14 @@ import { useBaseUrl } from '../../../../hooks/common/use-base-url'
 export const ShareModal: React.FC<ModalVisibilityProps> = ({ show, onHide }) => {
   useTranslation()
   const noteFrontmatter = useApplicationState((state) => state.noteDetails.frontmatter)
-  const editorMode = useApplicationState((state) => state.editorConfig.editorMode)
   const baseUrl = useBaseUrl()
   const noteIdentifier = useApplicationState((state) => state.noteDetails.primaryAddress)
 
   return (
     <CommonModal show={show} onHide={onHide} showCloseButton={true} title={'editor.modal.shareLink.title'}>
       <Modal.Body>
         <Trans i18nKey={'editor.modal.shareLink.editorDescription'} />
-        <CopyableField
-          content={`${baseUrl}n/${noteIdentifier}?${editorMode}`}
-          shareOriginUrl={`${baseUrl}n/${noteIdentifier}?${editorMode}`}
-        />
+        <CopyableField content={`${baseUrl}n/${noteIdentifier}`} shareOriginUrl={`${baseUrl}n/${noteIdentifier}`} />
```

Comment:
If you re-add the URL query parameters, they should not be removed here, otherwise a user might not know about them.

Reply:
After a short discussion I removed the URL query parameter feature completly because we couldn't construct a useful case where this is still needed.